### PR TITLE
Update card chooser part 2

### PR DESCRIPTION
**Pull request by FadhlanR**
- [x] Migrate card URL functionality to be part of the search box.
- [x] Add the "Create New [Card Type Display Name]" button in the card chooser modal. This button will open the edit view of a new item in the stack. If it is saved it will add a new card to the linksToMany field.

### CODE REVIEW

File: packages/host/tests/integration/components/operator-mode-test.gts [hunk 1]
```diff
@@ -1300,6 +1300,84 @@ module('Integration | operator-mode', function (hooks) {
...
+      'Woodster'
+    );
+    await click('[data-test-stack-card-index="1"] [data-test-save-button]');
+    await waitUntil(
+      () => !document.querySelector('[data-test-stack-card-index="1"]')
+    );
+    assert.dom('[data-test-field="friends"]').containsText('Woodster');
+  });
+
+  test('does not create a new card to add to a linksToMany field from card chooser, if user cancel the edit view', async function (assert) {
```
Comment (tintinthong) [inline packages/host/tests/integration/components/operator-mode-test.gts:1380]: I believe the card is created. But the card is just not linked to the parent card. Can you adjust the text to reflect the test.

Comment (FadhlanR) [reply packages/host/tests/integration/components/operator-mode-test.gts:1380]: The card is not created if the user doesn't click the save button. At least for now, I am unsure of the behavior after we implement the auto-save.

**Review by jurgenwerk — CHANGES_REQUESTED** [review]
I found bugs.

1. Placeholder
image

2. When I create a new Pet (for example), and save it, there are now 2 pets on the stack:
image

**Comment by FadhlanR** [comment]
@jurgenwerk I've fixed those two bugs even though the second bug is from the main branch.

**Comment by jurgenwerk** [comment]
Great!

**Review by jurgenwerk — APPROVED** [review]

**Review by burieberry — APPROVED** [review]

**Pull request merged**